lib/models/user_setting_model.dart

```diff
@@ -1,5 +1,5 @@
 class UserSetting {
```

Comment:
You have to add a new property here:

`bool taxInclusive;`

Reply:
What should be the default value?

Reply:
You don’t have to set a default, just like the `pushNotifications` property. Dart would assign `null` by default.